test/e2e/user-actions-benchmark.js

```diff
@@ -0,0 +1,142 @@
+const path = require('path');
+const { promises: fs } = require('fs');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { exitWithError } = require('../../development/lib/exit-with-error');
+const {
+  isWritable,
+  getFirstParentDirectoryThatExists,
+} = require('../helpers/file');
+const { convertToHexValue, withFixtures } = require('./helpers');
+
+const ganacheOptions = {
+  accounts: [
+    {
+      secretKey:
+        '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+      balance: convertToHexValue(25000000000000000000),
+    },
+  ],
+};
+
+async function loadNewAccount() {
+  let loadingTimes;
+
+  await withFixtures(
+    {
+      fixtures: 'imported-account',
+      ganacheOptions,
+    },
+    async ({ driver }) => {
+      await driver.navigate();
+      await driver.fill('#password', 'correct horse battery staple');
+      await driver.press('#password', driver.Key.ENTER);
+
+      await driver.clickElement('.account-menu__icon');
+      const timestampBeforeAction = new Date();
+      await driver.clickElement({ text: 'Create account', tag: 'div' });
+      await driver.fill('.new-account-create-form input', '2nd account');
+      await driver.clickElement({ text: 'Create', tag: 'button' });
+      await driver.waitForSelector({
+        css: '.currency-display-component__text',
+        text: '0',
+      });
+      const timestampAfterAction = new Date();
+      loadingTimes = timestampAfterAction - timestampBeforeAction;
+    },
+  );
+  return loadingTimes;
+}
+
+async function confirmTx() {
+  let loadingTimes;
+  await withFixtures(
+    {
+      fixtures: 'imported-account',
+      ganacheOptions,
+    },
+    async ({ driver }) => {
+      await driver.navigate();
+      await driver.fill('#password', 'correct horse battery staple');
+      await driver.press('#password', driver.Key.ENTER);
+
+      await driver.clickElement('[data-testid="eth-overview-send"]');
+
+      await driver.fill(
+        'input[placeholder="Search, public address (0x), or ENS"]',
+        '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+      );
+
+      const inputAmount = await driver.findElement('.unit-input__input');
+      await inputAmount.fill('1000');
+
+      await inputAmount.press(driver.Key.BACK_SPACE);
+      await inputAmount.press(driver.Key.BACK_SPACE);
+      await inputAmount.press(driver.Key.BACK_SPACE);
+      await driver.wait(async () => {
+        const sendDialogMsgs = await driver.findElements(
+          '.send-v2__form div.dialog',
+        );
+        return sendDialogMsgs.length === 1;
+      }, 10000);
+
+      await inputAmount.fill('1');
```

Comment:
Why do we enter an amount twice? 
It looks like first we enter 1000, clear the field, then enter 1.

Reply:
oh true! I removed the unnecessary steps (for context, this testcase was inherited from the send test file, which does this extra step, but not really needed here).
Thank you @PeterYinusa , fixed!